osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -52,20 +53,21 @@ func addCreateOsConfigTest(pkgTestSetup []*packageManagementTestSetup, testProje
 	for _, tuple := range tuples {
 		var oc *osconfigpb.OsConfig
 		var image string
+		uuid := utils.RandString(8)
```

Comment:
How about using a timestamp instead? Going down to millis should avoid duplicates and will make it easier to work backward from leaked resources.

Reply:
That's probably too long, we are limited here on resource name length and having the random bits be the thing that puts us over the limit is undesirable. Use utils.RandString(5) here instead. Also have this generated only once for the entire test suite (and add it to testName) so that we can easily associate test runs and resources (osconfigs and instances)

Reply:
Also I guess it may not be clear, the length issue is for compute resources, but I still feel strongly we should associate all resources created by the test. We could put timestamp in the osconfig description or something

Reply:
i initially used uuid(they have time stamp and chances of collision is very very low), but since they are 36 character long, we could have ended up with exceeding 64 characters for resource name.
I can change to use timestamp but then i would like to change the Randstring func instead of using timestamp just here, just want to keep it consistent throughout the application

Reply:
I'm fine with using a random slug and putting the timestamp in the description. Alternatively, having a timestamp based ID to associate with the entire test run would probably be ideal. Also, if we're using `RandString(5)` we'll want to call it something other than UUID.

Reply:
> Use utils.RandString(5) here instead. Also have this generated only once for the entire test suite (and add it to testName) so that we can easily associate test runs and resources (osconfigs and instances)

this totally makes sense.

I am going to change it to use 5 digit random string and use timestamp in the description